diff --git a/luci-app-adguardhome/root/etc/init.d/AdGuardHome b/luci-app-adguardhome/root/etc/init.d/AdGuardHome
index 651ecd9..76b688a 100755
--- a/luci-app-adguardhome/root/etc/init.d/AdGuardHome
+++ b/luci-app-adguardhome/root/etc/init.d/AdGuardHome
@@ -224,7 +224,7 @@ _do_redirect()
 		echo -n "0" > /var/run/AdGredir
 	fi
 
-	config_get configpath $CONFIGURATION configpath "/etc/AdGuardHome.yaml"
+	config_get configpath $CONFIGURATION configpath "/etc/config/AdGuardHome.yaml"
 	AdGuardHome_PORT="$(config_editor "dns.port" "" "$configpath" "1")"
 
 	[ ! -s "$configpath" ] && cp -f /usr/share/AdGuardHome/AdGuardHome_template.yaml "$configpath"
@@ -413,7 +413,7 @@ start_service() {
 	config_load "${CONFIGURATION}"
 
 	config_get hashpass $CONFIGURATION hashpass ""
-	config_get configpath $CONFIGURATION configpath "/etc/AdGuardHome.yaml"
+	config_get configpath $CONFIGURATION configpath "/etc/config/AdGuardHome.yaml"
 
 	if [ -n "$hashpass" ]; then
 		config_editor "users.password" "$hashpass" "$configpath"
diff --git a/packages/adguardhome/Makefile b/packages/adguardhome/Makefile
index ff603d4..c4f64e3 100644
--- a/packages/adguardhome/Makefile
+++ b/packages/adguardhome/Makefile
@@ -49,7 +49,7 @@ define Package/adguardhome/conffiles
 endef
 
 define Package/adguardhome/conffiles
-/etc/adguardhome/adguardhome.yaml
+/etc/config/adguardhome.yaml
 /etc/config/adguardhome
 endef
 
