diff --git a/luci-app-adguardhome/root/etc/init.d/AdGuardHome b/luci-app-adguardhome/root/etc/init.d/AdGuardHome
index 507ab25..44abdb1 100755
--- a/luci-app-adguardhome/root/etc/init.d/AdGuardHome
+++ b/luci-app-adguardhome/root/etc/init.d/AdGuardHome
@@ -214,7 +214,7 @@ _do_redirect() {
 	else
 		echo -n "0">/var/run/AdGredir
 	fi
-	config_get configpath $CONFIGURATION configpath "/etc/AdGuardHome.yaml"
+	config_get configpath $CONFIGURATION configpath "/etc/config/AdGuardHome.yaml"
 	AdGuardHome_PORT=$(config_editor "dns.port" "" "$configpath" "1")
 	if [ -z "$AdGuardHome_PORT" ]; then
 		AdGuardHome_PORT="0"
@@ -391,7 +391,7 @@ start_service() {
 	config_load "${CONFIGURATION}"
 	# update password
 	config_get hashpass $CONFIGURATION hashpass ""
-	config_get configpath $CONFIGURATION configpath "/etc/AdGuardHome.yaml"
+	config_get configpath $CONFIGURATION configpath "/etc/config/AdGuardHome.yaml"
 	[ -f $configpath ] && chmod 777 $configpath
 	if [ -n "$hashpass" ]; then
 		config_editor "users.password" "$hashpass" "$configpath"

diff --git a/packages/adguardhome/Makefile b/packages/adguardhome/Makefile
index d8961e3..f24a435 100644
--- a/packages/adguardhome/Makefile
+++ b/packages/adguardhome/Makefile
@@ -47,7 +47,7 @@ define Package/adguardhome
 endef
 
 define Package/adguardhome/conffiles
-/etc/adguardhome.yaml
+/etc/config/adguardhome.yaml
 /etc/config/adguardhome
 endef

