diff -Naur luci-app-adguardhome/Makefile luci-app-adguardhome/Makefile
--- luci-app-adguardhome/Makefile	2022-05-10 23:49:13.000000000 +0800
+++ luci-app-adguardhome/Makefile	2022-09-24 10:23:09.000000000 +0800
@@ -16,7 +16,7 @@
 define Package/$(PKG_NAME)/config
 config PACKAGE_$(PKG_NAME)_INCLUDE_binary
 	bool "Include Binary File"
-	default y
+	default n
 endef
 
 PKG_CONFIG_DEPENDS:= CONFIG_PACKAGE_$(PKG_NAME)_INCLUDE_binary
@@ -24,7 +24,7 @@
 define Package/luci-app-adguardhome/conffiles
 /usr/share/AdGuardHome/links.txt
 /etc/config/AdGuardHome
-/etc/AdGuardHome.yaml
+/etc/config/AdGuardHome.yaml
 endef
 
 define Package/luci-app-adguardhome/postinst
diff -Naur luci-app-adguardhome/root/etc/config/AdGuardHome luci-app-adguardhome/root/etc/config/AdGuardHome
--- luci-app-adguardhome/root/etc/config/AdGuardHome	2022-05-10 23:49:13.000000000 +0800
+++ luci-app-adguardhome/root/etc/config/AdGuardHome	2022-09-24 10:23:42.000000000 +0800
@@ -2,7 +2,7 @@
 	option enabled '0'
 	option httpport '3000'
 	option redirect 'none'
-	option configpath '/etc/AdGuardHome.yaml'
+	option configpath '/etc/config/AdGuardHome.yaml'
 	option workdir '/etc/AdGuardHome'
 	option logfile '/tmp/AdGuardHome.log'
 	option verbose '0'
diff -Naur luci-app-adguardhome/root/usr/share/AdGuardHome/links.txt luci-app-adguardhome/root/usr/share/AdGuardHome/links.txt
--- luci-app-adguardhome/root/usr/share/AdGuardHome/links.txt	2022-05-10 23:49:13.000000000 +0800
+++ luci-app-adguardhome/root/usr/share/AdGuardHome/links.txt	2022-09-24 10:24:57.000000000 +0800
@@ -1,3 +1,3 @@
-https://static.adguard.com/adguardhome/beta/AdGuardHome_linux_${Arch}.tar.gz
 https://github.com/AdguardTeam/AdGuardHome/releases/download/${latest_ver}/AdGuardHome_linux_${Arch}.tar.gz
+https://static.adguard.com/adguardhome/beta/AdGuardHome_linux_${Arch}.tar.gz
 https://static.adguard.com/adguardhome/release/AdGuardHome_linux_${Arch}.tar.gz
