diff --git a/luci-app-autotimeset/luasrc/controller/autotimeset.lua b/luci-app-autotimeset/luasrc/controller/autotimeset.lua
index da33581..c1f9321 100644
--- a/luci-app-autotimeset/luasrc/controller/autotimeset.lua
+++ b/luci-app-autotimeset/luasrc/controller/autotimeset.lua
@@ -5,14 +5,14 @@ function index()
 	if not nixio.fs.access("/etc/config/autotimeset") then
 		return
 	end
-        entry({"admin", "control"}, firstchild(), "Control", 44).dependent = false
-        local e = entry({"admin", "control", "autotimeset"}, alias("admin", "control", "autotimeset", "base"), _("Scheduled Setting"), 20)
+        entry({"admin", "services"}, firstchild(), "services", 44).dependent = false
+        local e = entry({"admin", "services", "autotimeset"}, alias("admin", "services", "autotimeset", "base"), _("Scheduled Setting"), 20)
 	e.dependent = false
 	e.acl_depends = { "luci-app-autotimeset" }
-        entry({"admin", "control", "autotimeset", "base"}, cbi("autotimeset/base"),  _("Scheduled Setting"), 1).leaf = true
-        entry({"admin", "control", "autotimeset", "log"}, form("autotimeset/log"), _("Log"), 2).leaf = true
-        entry({"admin","control","autotimeset","dellog"},call("dellog"))
-        entry({"admin","control","autotimeset","getlog"},call("getlog"))
+        entry({"admin", "services", "autotimeset", "base"}, cbi("autotimeset/base"),  _("Scheduled Setting"), 1).leaf = true
+        entry({"admin", "services", "autotimeset", "log"}, form("autotimeset/log"), _("Log"), 2).leaf = true
+        entry({"admin","services","autotimeset","dellog"},call("dellog"))
+        entry({"admin","services","autotimeset","getlog"},call("getlog"))
 end
 
 
diff --git a/luci-app-autotimeset/luasrc/view/autotimeset/log.htm b/luci-app-autotimeset/luasrc/view/autotimeset/log.htm
index 0f1b859..9fe1437 100644
--- a/luci-app-autotimeset/luasrc/view/autotimeset/log.htm
+++ b/luci-app-autotimeset/luasrc/view/autotimeset/log.htm
@@ -16,7 +16,7 @@ function createAndDownloadFile(fileName,content){
 }
 
 function apply_del_log(){
-	XHR.get('<%=url([[admin]],[[control]],[[autotimeset]],[[dellog]])%>',null,function(x,data){
+	XHR.get('<%=url([[admin]],[[services]],[[autotimeset]],[[dellog]])%>',null,function(x,data){
 	var lv=document.getElementById('cbid.logview.1.conf');
 		lv.innerHTML="";
 	}
@@ -35,7 +35,7 @@ function reverselog(){
 	return
 }
 
-XHR.poll(3,'<%=url([[admin]],[[control]],[[autotimeset]],[[getlog]])%>',null,
+XHR.poll(3,'<%=url([[admin]],[[services]],[[autotimeset]],[[getlog]])%>',null,
 	function(x,data){
 		var lv=document.getElementById('cbid.logview.1.conf');
 		lv.innerHTML=""
diff --git a/luci-app-autotimeset/po/zh-cn/autotimeset.po b/luci-app-autotimeset/po/zh-cn/autotimeset.po
index 9b322f0..1a79a07 100644
--- a/luci-app-autotimeset/po/zh-cn/autotimeset.po
+++ b/luci-app-autotimeset/po/zh-cn/autotimeset.po
@@ -11,6 +11,12 @@ msgstr "定时执行任务/开机启动任务设置"
 msgid "Scheduled Setting"
 msgstr "任务设置"
 
+msgid "Clear Log"
+msgstr "清理log"
+
+msgid "time Setting"
+msgstr "定时设置"
+
 msgid "Test/Verify Settings"
 msgstr "测试/验证设置"
 
