diff --git a/luci-app-cpufreq/Makefile b/luci-app-cpufreq/Makefile
index 5a95b05..cfde103 100644
--- a/luci-app-cpufreq/Makefile
+++ b/luci-app-cpufreq/Makefile
@@ -1,11 +1,11 @@
-# SPDX-License-Identifier: GPL-3.0-only
+﻿# SPDX-License-Identifier: GPL-3.0-only
 #
 # Copyright (C) 2021 ImmortalWrt.org
 
 include $(TOPDIR)/rules.mk
 
 LUCI_TITLE:=LuCI for CPU Freq Setting
-LUCI_DEPENDS:=@(arm||aarch64)
+#LUCI_DEPENDS:=@(arm||aarch64)
 
 PKG_NAME:=luci-app-cpufreq
 PKG_VERSION:=1
diff --git a/luci-app-cpufreq/root/etc/config/cpufreq b/luci-app-cpufreq/root/etc/config/cpufreq
index 5c2c070..62860b3 100644
--- a/luci-app-cpufreq/root/etc/config/cpufreq
+++ b/luci-app-cpufreq/root/etc/config/cpufreq
@@ -1,3 +1,7 @@
-
+﻿
 config settings 'cpufreq'
-
+	option maxfreq '716000'
+	option upthreshold '50'
+	option factor '10'
+	option minifreq '300000'
+	option governor 'ondemand'
\ No newline at end of file
diff --git a/luci-app-cpufreq/root/etc/uci-defaults/10-cpufreq b/luci-app-cpufreq/root/etc/uci-defaults/10-cpufreq
index c9dee88..6ab21d3 100755
--- a/luci-app-cpufreq/root/etc/uci-defaults/10-cpufreq
+++ b/luci-app-cpufreq/root/etc/uci-defaults/10-cpufreq
@@ -63,15 +63,15 @@ case "$DISTRIB_TARGET" in
 	"rockchip/armv8")
 		if echo "$CPU_POLICYS" | grep -q "4"; then
 			# RK3399
-			uci_write_config 0 schedutil 600000 1608000
-			uci_write_config 4 schedutil 600000 2016000
+			uci_write_config 0 schedutil 600000 1800000
+			uci_write_config 4 schedutil 600000 2208000
 		else
 			if echo "$CPU_FREQS" | grep -q "1992000"; then
 				# RK3568
 				CPU_MAX_FREQ="1992000"
 			else
 				# RK3328
-				CPU_MAX_FREQ="1512000"
+				CPU_MAX_FREQ="1800000"
 			fi
 			uci_write_config 0 schedutil 816000 $CPU_MAX_FREQ
 		fi
