diff --git a/luci-app-passwall2/Makefile b/luci-app-passwall2/Makefile
index 7d04e36..f115c08 100644
--- a/luci-app-passwall2/Makefile
+++ b/luci-app-passwall2/Makefile
@@ -101,7 +101,7 @@ config PACKAGE_$(PKG_NAME)_INCLUDE_Shadowsocks_Rust_Client
 	bool "Include Shadowsocks Rust Client"
 	depends on !i386
 	select PACKAGE_shadowsocks-rust-sslocal
-	default y if aarch64||x86_64
+	default y
 
 config PACKAGE_$(PKG_NAME)_INCLUDE_Shadowsocks_Rust_Server
 	bool "Include Shadowsocks Rust Server"
diff --git a/luci-app-passwall2/root/usr/share/passwall2/0_default_config b/luci-app-passwall2/root/usr/share/passwall2/0_default_config
index 647a2d9..8a67840 100644
--- a/luci-app-passwall2/root/usr/share/passwall2/0_default_config
+++ b/luci-app-passwall2/root/usr/share/passwall2/0_default_config
@@ -31,7 +31,7 @@ config global_forwarding
 	option udp_redir_ports '1:65535'
 	option accept_icmp '0'
 	option prefer_nft '1'
-	option tcp_proxy_way 'redirect'
+	option tcp_proxy_way 'tproxy'
 	option ipv6_tproxy '0'
 
 config global_xray
