diff --git a/packages/smartdns/Makefile b/packages/smartdns/Makefile
index d918c5b..58f81d8 100644
--- a/packages/smartdns/Makefile
+++ b/packages/smartdns/Makefile
@@ -6,13 +6,13 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=smartdns
-PKG_VERSION:=1.2020.30
-PKG_RELEASE:=1
+PKG_VERSION:=37.2
+PKG_RELEASE:=$(AUTORELEASE)
 
-PKG_SOURCE_PROTO:=git
-PKG_SOURCE_URL:=https://www.github.com/pymumu/packages/smartdns.git
-PKG_SOURCE_VERSION:=0aec326d551925a269a960677f4cd432d8e89385
-# PKG_MIRROR_HASH:=d5affc45a533e38ee04f3ce47b441aecf316cb9cb68ff410eede67090ec0fcc7
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://codeload.github.com/pymumu/packages/smartdns/tar.gz/Release$(PKG_VERSION)?
+PKG_HASH:=56819ebeb05a6921b6c5ccf27250e720c95eef314b30b408775df99768f1ce03
+PKG_BUILD_DIR:=$(BUILD_DIR)/$(PKG_NAME)-Release$(PKG_VERSION)
 
 PKG_MAINTAINER:=Nick Peng <pymumu@gmail.com>
 PKG_LICENSE:=GPL-3.0-or-later
