diff --git a/luci-app-ssr-plus/Makefile b/luci-app-ssr-plus/Makefile
index ecec17e..459139b 100644
--- a/luci-app-ssr-plus/Makefile
+++ b/luci-app-ssr-plus/Makefile
@@ -77,7 +77,7 @@ endchoice
 
 choice
 	prompt "Shadowsocks Client Selection"
-	default PACKAGE_$(PKG_NAME)_INCLUDE_Shadowsocks_NONE_Client
+	default PACKAGE_$(PKG_NAME)_INCLUDE_Shadowsocks_Rust_Client
 
 	config PACKAGE_$(PKG_NAME)_INCLUDE_Shadowsocks_NONE_Client
 	bool "None"
@@ -97,6 +97,8 @@ choice
 	bool "Shadowsocks Rust"
 endchoice
 
+menu "Configuration"
+
 choice
 	prompt "V2ray-core Selection"
 	default PACKAGE_$(PKG_NAME)_INCLUDE_Xray if aarch64||arm||i386||x86_64
@@ -168,6 +170,7 @@ config PACKAGE_$(PKG_NAME)_INCLUDE_ShadowsocksR_Libev_Server
 	bool "Include ShadowsocksR Libev Server"
 	default n
 
+endmenu
 endef
 
 
